README.md

```diff
@@ -61,8 +59,8 @@ These are all available inputs.
 ## Regional holidays
 
 This tool supports blocking merges based on regional holidays. For example, February 11th is "National Foundation Day"
-in Japan, and you can block merges on such a holiday via `prohibited-days-dates` with `H:Japan`. In addition, you can
-also block on the day before the regional holiday with `BH:Japan`.
+in Japan, you can block merges on such a holiday via `prohibited-days-dates` with `H:Japan`. In addition, you can
```

Comment:
Shouldn't we use a coordinating conjunction to join clauses in this context? I though "you can block merges on..." should be joined by a coordinating conjunction because the phrase is a clause.

Reply:
Yeah you are correct about that, too little sleep :D